ui/app/pages/create-account/import-account/private-key.js

```diff
@@ -9,9 +8,106 @@ import { DEFAULT_ROUTE } from '../../../helpers/constants/routes'
 import { getMetaMaskAccounts } from '../../../selectors/selectors'
 import Button from '../../../components/ui/button'
 
-PrivateKeyImportView.contextTypes = {
-  t: PropTypes.func,
-  metricsEvent: PropTypes.func,
+class PrivateKeyImportView extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+    metricsEvent: PropTypes.func,
+  }
+
+  static propTypes = {
+    importNewAccount: PropTypes.func.isRequired,
+    history: PropTypes.object.isRequired,
+    displayWarning: PropTypes.func.isRequired,
+    setSelectedAddress: PropTypes.func.isRequired,
+    firstAddress: PropTypes.object.isRequired,
```

Comment:
```suggestion
    firstAddress: PropTypes.string.isRequired,
```
This should always be a string; it's the address of the account, and it is derived from the key in the `accounts` object from the background.